packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -292,8 +297,39 @@ message Invitation {
 
     /// Guest should call `Authenticate` with the shared secret.
     SHARED_SECRET = 1;
+
+    /// Invitation is only claimable by a specific identity.
+    IDENTITY = 2;
   }
 
+  /// Randomly id generated by the host.
+  string invitation_id = 1;
+
+  /// Determines the behavior of the invitation.
+  Type type = 2;
+
+  /// Kind of access the invitation will grant.
+  Kind kind = 3;
+
+  /// How the invitation is authenticated.
+  AuthMethod auth_method = 4;
+
+  /// Beacon allows two peers to rendezvous on a p2p network.
+  Beacon beacon = 5;
+
+  optional google.protobuf.Timestamp expiry = 6;
```

Comment:
Should we include mention of the semantics around this? E.g. is this validated by the recipient, is it intended to be a security feature or just UI, etc.